src/ipc-handlers.ts

```diff
@@ -459,6 +471,17 @@ export async function clearAuthenticationToken() {
 	return oauthClient.clearAuthenticationToken();
 }
 
+export async function exportSite(
+	_event: IpcMainInvokeEvent,
+	options: ExportOptions
+): Promise< void > {
+	try {
+		await exportBackup( options );
+	} catch ( e ) {
+		console.log( e );
```

Comment:
Following other IPC handlers, it would be great to bubble up the error, as well as send it to Sentry.

```suggestion
		Sentry.captureException( e );
		throw e;
```